browser/resources/extensions/brave_item_list_more_items.js

```diff
@@ -5,7 +5,6 @@
 
 import {Polymer, html} from 'chrome://resources/polymer/v3_0/polymer/polymer_bundled.min.js'
```

Comment:
It might be tricky, but would be great to squash this commit into https://github.com/brave/brave-core/pull/4374/commits/0f85b567c91857ec2553e8a36a7077bea050413f

cc: @mkarolin